### PR TITLE
Added section on Setting Resource Quota for Extended Resources

### PR DESCRIPTION
**Pull request by ahardin-rh**
Resolves https://github.com/openshift/openshift-docs/issues/13447

Preview Build: http://file.rdu.redhat.com/~ahardin/01292019/quota-for-extended-resources/admin_guide/quota.html#extended-resources

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@jeremyeder PTAL

**Comment by jeremyeder** [comment]
this has the label branch/enterprise-3.11 but it is also valid for 4.0.

**Comment by ahardin-rh** [comment]
@jeremyeder Thanks! I will open a new PR to add this content to 4.0 docs.

**Comment by ahardin-rh** [comment]
@openshift/team-documentation Please peer review

**Comment by ahardin-rh** [comment]
@xingxingxia Can you please provide a QE review?

**Comment by ahardin-rh** [comment]
Note to self :spiral_notepad: Add this content to this WIP PR: https://github.com/openshift/openshift-docs/pull/13470

File: admin_guide/quota.adoc [hunk 1]
```diff
@@ -159,6 +159,126 @@ $ oc create quota <name> --hard=count/<resource>.<group>=<quota> <1>
 <1> `<resource>` is the name of the resource, and `<group>` is the API group, if applicable.
 Use the `kubectl api-resources` command for a list of resources and their associated API groups.
 
+[[extended-resources]]
+=== Setting Resource Quota for Extended Resources
+
+Overcommitment of resources is not allowed for extended resources, so you must
+specify `requests` and `limits` for the same extended resource in a quota.
```
Comment (mburke5678) [inline admin_guide/quota.adoc]: _Quota items... is allowed_ sounds incorrect. _are allowed_? In line 167

**Review by mburke5678 — COMMENTED** [review]
@ahardin-rh one nit. Otherwise LGTM.

**Pull request merged**